VMEncryption/main/DiskUtil.py

```diff
@@ -635,6 +635,10 @@ def get_mount_items(self):
 
         return items
 
+    def is_in_memfs_root(self):
+        mounts = file('/proc/mounts', 'r').read()
+        return bool(re.search(r'/\s+tmpfs', mounts))
```

Comment:
corner case - if any standard mount point has the string "tmpfs" in the name this will erroneously return true.  If you agree, it may be worth adding this as a todo comment as there may be a better approach in the future.

Reply:
I think '\s+' in regex-speak means "atleast one space-type character".
and because it is rare for mount points to have a bunch of spaces followed by "tmpfs" I feel its okay to leave it here for now.
It should definitely be made more robust later though.
Will add the TODO.